lib/utils/screen-manager.js

```diff
@@ -19,6 +19,7 @@ var ScreenManager = module.exports = function (rl) {
   this.extraLinesUnderPrompt = 0;
 
   this.rl = rl;
+  if (rl) rl.activeScreen = this;
```

Comment:
Gotcha. Though it's difficult to do this cleanly with the current class hierarchy.